### PR TITLE
FOUR-14799: Improve error messaging for AI 

### PR DESCRIPTION
**Pull request by agustinbusso**
## Issue & Reproduction Steps
Descripción

In sync with @Claudia Iriarte some modifications in the error messages are required:

Instead of showing the Server error alert message in all the pages all time when AI micro service is down, show the message in the console as error.

We need to show a more user friendly alert message ProcessMaker AI is currently offline. Please try again later. just when the user performs an specific AI action like:

- Clicks on generate process from text button
- Clicks on generate process from image button
- Clicks on generate script from text button
- Clicks on document script button
- Clicks on clean script button
- Clicks on explain script button
- Clicks on generate process assets button in modeler
- Clicks on generate form button in screen builder

By the other hand we need an informational tooltip message in the AI modeler component, similar to the following screen (This is an AI feature and can provide …):

image (1)

## Related Tickets and PRS
[FOUR-14799](https://processmaker.atlassian.net/browse/FOUR-14799)
- [core PR](https://github.com/ProcessMaker/processmaker/pull/6579)
- [package-ai PR](https://github.com/ProcessMaker/package-ai/pull/139)
- [modeler PR](https://github.com/ProcessMaker/modeler/pull/1807)

[FOUR-14799]: https://processmaker.atlassian.net/browse/FOUR-14799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ci:next

### CODE REVIEW

File: resources/js/processes/scripts/components/AiTab.vue [hunk 1]
```diff
@@ -353,8 +353,8 @@ export default {
             this.$emit("request-started", this.progress, this.$t("Cleaning"));
           }
         })
-        .catch((error) => {
-          const errorMsg = error.response?.data?.message || error.message;
+        .catch(() => {
+          const errorMsg = this.$t("ProcessMaker AI is currently offline. Please try again later.");
```
Comment (danloa) [inline resources/js/processes/scripts/components/AiTab.vue:357]: I don't see this translation in en.json

File: resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue [hunk 2]
```diff
@@ -108,7 +108,7 @@ export default {
           this.tokens = response.data.tokens;
         }).catch((error) => {
           const errorMsg = error.response?.data?.message || error.message;
-          window.ProcessMaker.alert(errorMsg, "danger");
+          console.error(errorMsg);
```
Comment (danloa) [inline resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue:111]: better to use console.error

File: resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue [hunk 3]
```diff
@@ -140,7 +140,7 @@ export default {
         }).catch((error) => {
           if (error.response.status !== 404) {
             const errorMsg = error.response?.data?.message || error.message;
-            window.ProcessMaker.alert(errorMsg, "danger");
+            console.log(errorMsg);
```
Comment (danloa) [inline resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue:143]: better to use console.error

File: resources/js/processes/translations/components/ProcessTranslationListing.vue [hunk 4]
```diff
@@ -292,7 +292,7 @@ export default {
             this.promptSessionId = '';
             this.fetchHistory();
           } else {
-            window.ProcessMaker.alert(errorMsg, 'danger');
+            console.error(errorMsg);
```
Comment (danloa) [inline resources/js/processes/translations/components/ProcessTranslationListing.vue:295]: better to use console.error

File: resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue (hunk 2 above)
Comment (agustinbusso) [reply resources/js/processes/scripts/components/GenerateScriptTextPrompt.vue:111]: This is already using console.error

File: resources/js/processes/translations/components/ProcessTranslationListing.vue (hunk 4 above)
Comment (agustinbusso) [reply resources/js/processes/translations/components/ProcessTranslationListing.vue:295]: This is already using console.error

File: resources/js/processes/scripts/components/AiTab.vue (hunk 1 above)
Comment (agustinbusso) [reply resources/js/processes/scripts/components/AiTab.vue:357]: Added to translation file

**Pull request merged**